packages/core/src/lib/handler/email.handler.ts

```diff
@@ -36,7 +36,21 @@ export class EmailHandler {
     } else {
       html = await this.message.template(templatePayload);
```

Comment:
Can we compile the `template` key if a function was passed instead of a string?

Reply:
Can you clarify? Maybe give some examples?!

Reply:
Currently if in the `IMessage` the `template` key was passed as function, we are bypassing our handlebars compiler.
I want it to be consistent with the subject behaviour, so we can make after this line

```javascript
html = compileTemplate(html);
```